lib/build.command.js

```
@@ -9,36 +9,35 @@
 module.exports = function (Aquifer) {
   'use strict';
 
+  // If the current directory doesn't contain an aquifer project, exit.
+  if (!Aquifer.initialized) {
+    return;
```

@patrickocoffeyo Should this throw an error instead of just returning?